packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -14,23 +14,28 @@ export 'package:flutter/foundation.dart' show DiagnosticsNode;
 
 /// Signature for the callback passed to the [Ticker] class's constructor.
 ///
-/// The argument is the time that the object had spent enabled so far
-/// at the time of the callback being called.
+/// The argument is the time elapsed from
+/// the frame timestamp when the ticker was last started
+/// to the current frame timestamp.
 typedef TickerCallback = void Function(Duration elapsed);
 
 /// An interface implemented by classes that can vend [Ticker] objects.
 ///
+/// To obtain a [TickerProvider], consider mixing in either
+/// [TickerProviderStateMixin] (which always works)
+/// or [SingleTickerProviderStateMixin] (which is more efficient when it works)
+/// to make a [State] subclass implement [TickerProvider].
+/// You might then pass that [State] to lower-level widgets
+/// or other related objects.
```

Comment:
Might as well do some rewording here to follow the [style guide](https://github.com/flutter/flutter/blob/58068d8ea4eca8a17d99469697aaff46255522e9/docs/contributing/Style-guide-for-Flutter-repo.md#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple).

```suggestion
/// The [State], or its constituent [AnimationController]s,
/// might then be passed to lower-level widgets or other related objects.
```

Reply:
Thanks, yeah. Rewritten like this:
```suggestion
/// That [State] can then be passed to lower-level widgets
/// or other related objects.
```

The [AnimationController]s might also be passed down; but that wouldn't be a way of obtaining a [TickerProvider], which is what this paragraph is about.